build/webpack/modern/webpack.modern.base.cjs

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const { merge } = require('webpack-merge');
+const { commonBaseConfig } = require('../common/webpack.common.base.cjs');
+
+const modernConfig = merge(commonBaseConfig, {
+    target: ['browserslist']
+});
+
+modernConfig.module.rules[0].use.push({
+    loader: 'babel-loader',
+    options: {
+        sourceType: 'unambiguous',
+        presets: [
+            [
+                '@babel/preset-env',
+                {
+                    useBuiltIns: 'usage',
+                    corejs: '3.39.0',
+                }
+            ],
+        ],
+        plugins: [
+            '@babel/plugin-transform-runtime',
+            '@babel/plugin-transform-parameters'
+        ],
+    },
+},)
+
+const umdConfig = merge(modernConfig, {
+    output: {
+        path: path.resolve(__dirname, '../../../dist/modern/umd'),
+        publicPath: '/dist/modern/umd/',
+        library: 'dashjs',
+        libraryTarget: 'umd',
+        libraryExport: 'default'
+    },
+});
```

Comment:
The `legacy` version is already a UMD bundle. What is the advantage of having a modern non-ESM variant? I would think that anyone needing to include the library via script/src would use legacy+umd and anyone in a modern browser, or using a build system, would use modern+esm.